README.md

```diff
@@ -43,18 +43,15 @@ ARGUMENTS
 ```
   Usage: 
 
-      easy-split-multi-deposit.sh [{--springfield-inbox|-s} <dir>] [{--staging-dir|-d} <dir>] <multi-deposit-dir> <output-deposits-dir> <datamanager>
+      easy-split-multi-deposit.sh [{--staging-dir|-s} <dir>] <multi-deposit-dir> <output-deposits-dir> <datamanager>
 
   Options:
 
-    -s, --springfield-inbox  <arg>   The inbox directory of a Springfield Streaming Media Platform installation.
-                                     If not specified the value of 'springfield-inbox' in
-                                     'application.properties' is used. (default = data/springfield-inbox)
-    -d, --staging-dir  <arg>         A directory in which the deposit directories are created, after which they
-                                     will be moved to the 'deposit-dir'. If not specified, the value of
-                                     'staging-dir' in 'application.properties' is used. (default = data/staging)
-        --help                       Show help message
-        --version                    Show version of this program
+  -s, --staging-dir  <arg>   A directory in which the deposit directories are created, after which they will
+                             be moved to the 'deposit-dir'. If not specified, the value of 'staging-dir' in
```

Comment:
is deposit-dir equal to output-deposits-dir? why then two names? let's opt for 'output-deposit-dir' to clearly state the intent (also r60)